types/userFutureStatus.d.ts

```diff
@@ -0,0 +1,14 @@
+import { userState, statusState } from "../constants/userStatus";
+
+export type UserFutureStatusType = {
+  id?: string;
+  requestId?: string;
+  status: userState.OOO | userState.IDLE | userState.ACTIVE;
+  state: statusState.UPCOMING | statusState.APPLIED | statusState.NOT_APPLIED;
+  from: number;
+  endsOn?: number;
+  userId: string;
+  message?: string;
+  updatedAt?: number;
```

Comment:
why there is an updatedAt field?

Reply:
When this doc is updated it can be helpful

Reply:
According to the implementation, the doc never updates. It recreates a new doc.

Reply:
Removed